# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xf86-video-fbdev/hotfix.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

From 4730f13b9261a00b87b3e98f070d100d8d72bef7 Mon Sep 17 00:00:00 2001
From: Tuomo Rinne <tuomo.rinne@pm.me>
Date: Mon, 14 Jan 2019 22:33:42 +0000
Subject: Fix build error when building with xserver built using meson

---
 src/fbdev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/fbdev.c b/src/fbdev.c
index f25ef72..e4f66a2 100644
--- a/src/fbdev.c
+++ b/src/fbdev.c
@@ -1010,7 +1010,7 @@ FBDevScreenInit(SCREEN_INIT_ARGS_DECL)
 	fPtr->CloseScreen = pScreen->CloseScreen;
 	pScreen->CloseScreen = FBDevCloseScreen;
 
-#if XV
+#ifdef XV
 	{
 	    XF86VideoAdaptorPtr *ptr;
 
-- 
cgit v1.2.1


